connectivity/CMakeLists.txt

```diff
@@ -22,18 +21,18 @@ add_library(mbed-nanostack-sal_stack INTERFACE)
 add_library(mbed-nanostack-sal_stack-event_loop INTERFACE)
 add_library(mbed-nanostack-libservice INTERFACE)
 add_library(mbed-netsocket INTERFACE)
-add_library(mbed-nfc INTERFACE)
 add_library(mbed-ppp INTERFACE)
 add_library(mbed-wifi INTERFACE)
 
 
 add_subdirectory(FEATURE_BLE)
 add_subdirectory(cellular)
-add_subdirectory(drivers)
 add_subdirectory(libraries)
 add_subdirectory(lorawan)
 add_subdirectory(lwipstack)
 add_subdirectory(mbedtls)
 add_subdirectory(nanostack)
 add_subdirectory(netsocket)
 add_subdirectory(nfc)
+
+add_subdirectory(drivers)
```

Comment:
ble PR got merged, if this is rebased, it should resolve the conflict

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/9a15a27e297662fa54890a29e15c06ca72151a75..188f7e7956fb5bdcde4fa0d47cacc986cb22df4d).